### DISCUSSION TITLE
2 questions about your paper template in Obsidian

### DISCUSSION THREAD

**Discussion by Kdest14** [BryanJenksCommunity/FAQ#176] [Q&A] [answered]
I was looking this video:  https://www.youtube.com/watch?v=0hFJw0kEHlU&t=1052s 

I want to have the same colors of my notes in Zotero when I extracted them to Obsidian as you have in your template and also have the link to the page in the PDF in Zotero. For some reason it doesn't works. I know something miss with the code for the link to the page, but I have no idea what is the problem with the colors. When I import my notes from Zotero Obsidian recognize the colors in the code but when I go to the read mode it just show me the default color that I define for the highlighted text background with the Style plugin. 
![image](https://github.com/BryanJenksCommunity/FAQ/assets/52632534/84c511a9-fa3e-4d67-b8ef-91d6e9b5cb3f)

![image](https://github.com/BryanJenksCommunity/FAQ/assets/52632534/eca16873-879d-482e-9dfd-7b580d4bb2fa)

So maybe I miss some steps to show the correct color of the original notes in Zotero?

**Comment by tallguyjenks** [comment, marked as answer]
i have different highlighting colors defined in my theme's CSS file, so that is what youre missing

**Reply by Kdest14** [reply]
Thanks
